satellite/audit/reservoir.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"math/rand"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Reservoir holds a certain number of segments to reflect a random sample
+type Reservoir struct {
+	Paths   []storj.Path
+	Size    int
+	NumSeen int
```

Comment:
I don't immediately associate `NumSeen` with the index of the reservoir. How about using `index` instead of `NumSeen`. Also, I think it does not need to be exported as it a rather internal thing for the reservoir, i.e. `index` instead of `Index`.